src/errors/test-run/templates.js

```diff
@@ -293,5 +293,11 @@ export default {
         An unhandled error occurred in the "${err.methodName}" method of the "${err.hookClassName}" class:
         
         ${escapeHtml(err.errMsg)}
+    `),
+
+    [TEST_RUN_ERRORS.executeNodeExpressionError]: err => markup(err, `
+        An unhandled error "${escapeHtml(err.errMsg)}" occurred in the code step:
+        ${escapeHtml(err.expression)}
+        at (${err.line}:${err.column})
```

Comment:
BTW, we have yet to decide how to exactly call these 'code steps'. (I'm not sure `Node.js Code` from the prototype is okay)

Reply:
May be we can use some neutral message here, that does not depend on TestCafe Studio definitions

Reply:
This is what the correctors approved:

```js
    [TEST_RUN_ERRORS.executeJsExpressionError]: err => markup(err, `
        An unhandled error occurred in a step with custom JS code:

        ${escapeHtml(err.errMsg)}

        ${escapeHtml(err.expression)}
        at ${err.line}:${err.column}
    `)
```

Is is okay?

Reply:
it is